docs/User-Guide_Advanced-Configuration.md

```diff
@@ -0,0 +1,406 @@
+make
+make install
+```
+
+??? "Build log"
+
+    ```text
+    make ARCH=arm64 CROSS_COMPILE= -C /lib/modules/6.6.62-current-sunxi64/build M=/root/8821au-20210708  modules
+    make[1]: Entering directory '/usr/src/linux-headers-6.6.62-current-sunxi64'
+      CC [M]  /root/8821au-20210708/core/rtw_cmd.o
+      CC [M]  /root/8821au-20210708/core/rtw_security.o
+      CC [M]  /root/8821au-20210708/core/rtw_debug.o
+      CC [M]  /root/8821au-20210708/core/rtw_io.o
+      CC [M]  /root/8821au-20210708/core/rtw_ioctl_query.o
+      CC [M]  /root/8821au-20210708/core/rtw_ioctl_set.o
+      CC [M]  /root/8821au-20210708/core/rtw_ieee80211.o
+      CC [M]  /root/8821au-20210708/core/rtw_mlme.o
+      CC [M]  /root/8821au-20210708/core/rtw_mlme_ext.o
+      ...
+      [ goes on for a while ]
+      ...
+      LD [M]  /root/8821au-20210708/8821au.o
+      MODPOST /root/8821au-20210708/Module.symvers
+      CC [M]  /root/8821au-20210708/8821au.mod.o
+      LD [M]  /root/8821au-20210708/8821au.ko
+    make[1]: Leaving directory '/usr/src/linux-headers-6.6.62-current-sunxi64'
+    ```
+
+Then attempt to load the driver and check the `dmesg` output.
+
+```sh
+insmod 8821au.ko
+usbcore: registered new interface driver rtl8821au
+```
+
+Plug the USB wireless adaptor and proceed with the [network configuration](User-Guide_Networking.md).
+
+
+## Downgrade a kernel package with APT
+
+Sometimes, it can be necessary to downgrade a package version, e.g. to fall back to a previous kernel version.
+
+```sh
+apt install linux-image-sun8i=5.13
+```
+
+This example is for the H3 legacy kernel. Check [this page](https://www.armbian.com/kernel/) for others.
+
+!!! danger
+
+    Please note that this situation or task is very uncommon. Version dependencies between packages can create serious conflicts when attempting a downgrade. If you force anything here, you can easily destroy your system beyond the point of repair. 
+
+
+## Toggle boot output
+
+[Boot parameters](http://redsymbol.net/linux-kernel-boot-parameters/) are edited or changed directly in `/boot/boot.cmd` (**not recommended**) or via variables in `/boot/armbianEnv.txt`:
+
+```diff
+- console=both
++ console=serial
+```
+
+To disable the console entirely (also **not recommended** and only as an example) one would set `console=none`.
+
+To recompile `boot.cmd` to `boot.scr` if it was changed:
+
+```sh
+mkimage -C none -A arm -T script -d /boot/boot.cmd /boot/boot.scr
+```
+
+And reboot.
+
+The serial console on imx6 boards is `ttymxc0` (Hummingboard, Cubox-i) or `ttymxc1` (Udoo).
+
+
+## Toggle boot verbosity
+
+<!-- TODO: Isn't this better suited for Troubleshooting; at least cross-link from there here -->
+
+Using Armbian from version 5.05 to 5.20, you have to touch/delete `/boot/.force-verbose` to increase the boot verbosity.
+
+With more recent Armbian builds, you have to alter the `verbosity=` line in `/boot/armbianEnv.txt` which defaults to `1` (which means less verbose) and has a maximum value of `7`.
```

Comment:
"With more recent ... ". This is now default, rephrase.

Reply:
It is the default now.